.github/workflows/master_merge.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           if [ -f convoy.yaml ]; then
             if [[ "${{ github.event.pull_request.base.ref }}" == "main" ]]; then
               APP_GROUP=$(yq '.appGroup // "tessell"' convoy.yaml)
-              URL="http://${{vars.CONVOY_API_ENDPOINT}}/devops/applications/app-groups/$APP_GROUP/latest-main-release-label"
+              URL="http://${{vars.CONVOY_DEV_API_ENDPOINT}}/devops/applications/app-groups/$APP_GROUP/latest-main-release-label"
```

Comment:
Switch to `CONVOY_API_ENDPOINT`

Can we have a env variable as `CONVOY_API_ENDPOINT` -> This env variable will always point to `vars.CONVOY_API_ENDPOINT`. Only during testing we switch to dev. This way there is single point of change

Reply:
Yes I'll make the changes